pulser-core/pulser/backend/remote.py

```diff
@@ -161,3 +173,34 @@ def _type_check_job_params(job_params: list[JobParams] | None) -> None:
                     "All elements of 'job_params' must be dictionaries; "
                     f"got {type(d)} instead."
                 )
+
+    def open_batch(self) -> Self:
+        """Create an open batch that can continue recieve new job.
+
+        As long as jobs are submitted within an open context
+        manager.
+
+        Returns:
+            A class instance with an associated batch_id property
+        """
+        # Create batch and receive submission id
+        submission = cast(
+            RemoteResults, self._connection.submit(self._sequence, open=True)
+        )
```

Comment:
I guess this creates an empty batch because there are no `job_params`, right? I'm assuming that's ok for the SDK?

Reply:
So I know it's ok for the API. Will confirm the SDK has no issue with it

Reply:
Just to close this thread, this will be fine if the branch is `open`. Then we can add jobs after

Reply:
Ok, so `pasqal-cloud` supports opening an empty batch after all, or is that features being added?